nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/summary/ui/connection-status-listing/connection-status-listing.component.ts

```diff
@@ -104,4 +108,15 @@ export class ConnectionStatusListing {
             })
         );
     }
+
+    openSystemDiagnostics(event: MouseEvent) {
+        event.stopPropagation();
```

Comment:
I don't think this `stopPropagation` is needed here. It's needed in table actions which results in the user being navigated someplace. That navigation conflicted with the row selection which also changed the route. This comment applies to all instances of `openSystemDiagnostics` in all component status listings.